pkgs/development/libraries/tdlib/default.nix

```diff
@@ -0,0 +1,23 @@
+{ fetchFromGitHub, gperf, openssl, readline, zlib, cmake, stdenv }:
+
+stdenv.mkDerivation rec {
+  version = "1.4.0";
+  pname = "tdlib-${version}";
```

Comment:
when `pname` and `version` are used `mkDerivation` automatically sets `name = ${pname}-${version}` and so this line should just be `pname = "tdlib";`

Reply:
Thank you!